bin/mssql

```diff
@@ -61,8 +62,8 @@ async function resolveConfig (opts, cfgFile) {
     ...config,
     ...cfg,
     options: {
+      ...cfg.options,
       ...(config.options || {}),
-      ...cfg.options
```

Comment:
Question: How come the order of these probs have changed?

The impact of this change will be that CLI options will be overridden by the config file; I think the explicit CLI argument should override the config file, but regardless this is a change of behaviour that makes this a "breaking change" IMO and so we should really avoid it just to minimise disruption upstream.

Reply:
I've spent some time thinking about this and I agree that the CLI should prevail.

But, by removing the default value from `--tds-version` on commander, it returns back `undefined`. So even if I defined the `tds-version` in a config file it gets overwritten.

I think we should further improve this by merging both when having valid values and make CLI args always prevail.